lib/modules/manager/pep621/readme.md

```diff
@@ -0,0 +1,11 @@
+This manager supports updating dependencies inside of `pyproject.toml` files.
+
+Outside standard dependencies, following toolsets are supported:
```

Comment:
```suggestion
In addition to standard dependencies, the following toolsets are also supported:
```

Maybe we can use `In addition to` and `also` to clarify that we support standard dependencies _and_ the listed toolsets.

Reply:
To late, needs another PR 🙃